### ISSUE TITLE
Can you stop at the high level node_modules?

### ISSUE THREAD

**Issue by Poimen** [voidcosmos/npkill#139]
**Is your feature request related to a problem? Please describe.**
I have a lot of mono-repo style directories, but probably would apply other times, and there are `node_modules` nested inside `node_modules`. For instance:
```
C:\git/website-labs/reactjs/responsive-navbar-1/node_modules
C:\git/website-labs/reactjs/responsive-navbar-1/node_modules/chokidar/node_modules
```

The subsequent `node_modules` is not based on "my" project but included by other packages.

**Describe the solution you'd like**
It would be great to include a command line option that would stop at the high level/first `node_modules` it encountered. So the output would only show:
```
C:\git/website-labs/reactjs/responsive-navbar-1/node_modules
```

**Describe alternatives you've considered**
Start the `npkill`, delete the higher directory, quit `npkill` and restart

**Comment by zaldih** [comment]
Hi! Thanks for opening the Issue.

It is indeed a bug introduced in the latest version (0.10.0) that only affects Windows.
I just noticed it a few hours ago while working on something else.

When #136 is merged this will be fixed.
For now I suggest using an older version like  `npx npkill@0.8.3`

**Comment by zaldih** [comment]
It should be fixed in v0.11.0 🎉
